testing/tests/DevExpress.ui.widgets.form/form.tests.js

```diff
@@ -4032,6 +4032,25 @@ function getColsCountFromDOM($form) {
     return result;
 }
 
+QUnit.test('Setting "colCountByScreen" property of group with dot notation should update layout', function(assert) {
+    const $form = $('#form').dxForm({
```

Comment:
We can get instance of the form it right away as follows
```
const form = $('#form').dxForm({}).dxForm('instance');
```
To get $element of the Form use $element() method as follows
```
assert.strictEqual(getColsCountFromDOM(form.$element()), 2);
```
JYI. It allows to write less code in tests.